sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md

```diff
@@ -20,13 +20,179 @@ rename-mapping:
   RecommendationType.IoT_ACRAuthentication: IotAcrAuthentication
   RecommendationType.IoT_IPFilter_DenyAll: IotIPFilterDenyAll
   RecommendationType.IoT_IPFilter_PermissiveRule: IotIPFilterPermissiveRule
+  DefenderForServersAwsOfferingVaAutoProvisioning: DefenderForServersAwsOfferingVulnerabilityAssessmentAutoProvisioning
+  DefenderForServersAwsOfferingVaAutoProvisioningConfiguration: DefenderForServersAwsOfferingVulnerabilityAssessmentAutoProvisioningConfiguration
```

Comment:
Is this too long...

Reply:
refer the bellows, how about trim the prefix `DefenderForServers` ?

Reply:
Below case is to remove a duplicate `DefenderForServers` while here we cannot remove `DefenderForServers` prefix as there are also similar models prefixed with `DefenderForDatabases` or `DefenderForContainers`.